src/main/java/zosjobs/input/DeleteJobParams.java

```diff
@@ -0,0 +1,74 @@
+package zosjobs.input;
+
+import java.util.Optional;
+
+public class DeleteJobParams {
+
+    private Optional<String> jobname;
+    private Optional<String> jobId;
+    private Optional<String> modifyVersion;
+
+    public DeleteJobParams(DeleteJobParams.Builder builder) {
+
+
+        if (builder.jobname != null)
+            this.jobname = Optional.ofNullable(builder.jobname);
+        else this.jobname = Optional.empty();
+
+        if (builder.jobId != null)
+            this.jobId = Optional.ofNullable(builder.jobId);
+        else this.jobId = Optional.empty();
+
+        if (builder.modifyVersion != null)
+            this.modifyVersion = Optional.ofNullable(builder.modifyVersion);
+        else this.modifyVersion = Optional.empty();
+    }
+
+
+    public Optional<String> getJobname() {
+        return jobname;
+    }
+
+    public Optional<String> getJobId() {
+        return jobId;
+    }
+
+    public Optional<String> getModifyVersion() { return modifyVersion; }
+
+    @Override
+    public String toString() {
+        return "DeleteJobParms{" +
+                "jobId=" + jobId +
+                ", jobname=" + jobname +
+                ", modifyVersion=" + modifyVersion +
+                '}';
+    }
+
+    public static class Builder {
```

Comment:
do we need to use builder pattern for this class that only has three members might be an overkill. seems all three members are required so not sure if optional builders are needed.. what do you think? use this without builder but with Optional.. are all three members are required?

Reply:
Hey Frank, I'm not clear about what I need to do here. I've resolved other comments and was hoping you could specify a bit more on what I should do in this? I'm currently marking this as resolved but feel free to unresolve it with a comment maybe on how should solve it. Please review the code again if there's anything else I should do as well.